### ISSUE TITLE
React left-nav crowds layout

### ISSUE THREAD

**Issue by TomNUSDS** [CDCgov/prime-reportstream#2866]
**Describe the bug**
The Left Nav is a duplicate of SOME the Top Nav. This is fine for desktop layout, but narrow and mobile layouts get crowded.

You can see that the navigation is duplicated from the "Menu" and the Left Nav.
<img src="https://images.zenhubusercontent.com/6001ffdbfa03b2340a4ca394/17fa1fb5-8bcd-45d6-a66e-8b82f040150c" width="400px">

For mobile, we hide the Left Nav, then the content section can fill the whole width:
<img src="https://images.zenhubusercontent.com/6001ffdbfa03b2340a4ca394/5a0391e6-7655-4f68-86f6-81c7846c7d75" width="400px">

**Comment by TomNUSDS** [comment]
@drew-usds Feel free to close with "not a bug", but it does seem odd to me.

**Comment by loripusey** [comment]
@TomNUSDS can you tell us what screen size does this bug affect? tablets? phone-size only?

**Comment by loripusey** [comment]
@TomNUSDS ?

**Comment by TomNUSDS** [comment]
It happens at all sizes for desktop (as small as it can be resized before it hits the minimum size limit). 

Mobile does something different but it's just as annoying?
![image](https://user-images.githubusercontent.com/74203452/141830203-bef64cc1-86c2-40de-b52e-4f7bb86996a6.png)

But this is a low priority and is ultimately a decision **design** should make? 

My recommendation is that we hide this left-nav when the top mobile "Menu" button appears.

**Comment by loripusey** [comment]
@drew-usds can you make some recommendations for this? Thanks.

**Comment by drew-usds** [comment]
Yeah totally hear y'all on this, was a known problem when we rolled it out.

Have been working on some new IA ideas on in-progress content-related cards. Will update this issue when we have some more solid ideas on those issues. This might just get resolved when we roll out some updated IA.

**Comment by loripusey** [comment]
@drew-usds Can you confirm if this problem still exists?

**Comment by drew-usds** [comment]
@loripusey Yes still a problem, but not a high priority based on everything else we've got. This is a piece of larger problems we want to address around nav you get based on auth status, user type etc. 

But in the meantime some futzy sidenavs are OK. We can push this down the priority ladder now but should leave it in the backlog and definitely eventually get to.

**Comment by kevinhaube** [comment]
Only page still using this model would be Product as of Dec 6 2022; would need a new page design to fix this.

**Comment by chris-kuryak** [comment]
Closing because this is no longer an issue.
